README.md

```diff
@@ -9,11 +9,15 @@ This project shows how Fable can be used fullstack with a node.js server written
 - [Mono Framework](https://www.mono-project.com/download/stable/) for some tooling if working in non-Windows environment
 - An F# IDE, like Visual Studio Code with [Ionide extension](http://ionide.io/)
 
-## Installing dependencies
+## Installing & running (in dev mode)
```

Comment:
There's a `Development` section just below that already mentions the `npm run start` command.